src/main/webapp/js/adminEmail.js

```diff
@@ -1,5 +1,7 @@
-var callbackFunction;
+// Form input placeholders
+var PLACEHOLDER_IMAGE_UPLOAD_ALT_TEXT = 'Please enter an alt text for the image';
```

Comment:
I thought "My alt text" is not very informative (see the screenshot above).
A call-to-action may prompt the admin to actually enter a proper alt text.